src/tensors/cpu/tensor_operators.cpp

```diff
@@ -707,7 +719,7 @@ void CrossEntropyPickBackward(Tensor out_,
 
     // cross-entropy
     for(int i = 0; i < cols; ++i) {
-      float sub = (float)(i == (int)pick[j]);
+      float sub = (float)(i == (int)pick_->data<IndexType>()[j]);
```

Comment:
Here it really looks like `pick_` is a class member.

Reply:
Brrr. As above.